AODN/AODN-WAVE-DM/DOT-WA-WAVE/waverider_library/common_waverider.py

```diff
@@ -303,30 +305,33 @@ def set_var_attr(nc_file_obj, var_mapping, nc_varname, df_varname_mapped_equival
     :return:
     """
 
-    setattr(nc_file_obj[nc_varname], 'units', var_mapping.loc[df_varname_mapped_equivalent]['UNITS'])
+    if not pd.isnull(var_mapping.loc[df_varname_mapped_equivalent]['STANDARD_NAME']):
+        setattr(nc_file_obj[nc_varname], 'standard_name',
+                var_mapping.loc[df_varname_mapped_equivalent]['STANDARD_NAME'])
```

Comment:
I think you would like to add the COMMENT one too since you added it to the csv file